Dockerfile

```diff
@@ -1,5 +1,44 @@
 ###################
-# STAGE 1: builder
+# STAGE 1.1: builder frontend
+###################
+
+FROM node:15.8.0-alpine as frontend
+
+WORKDIR /app/source
+
+ENV FC_LANG en-US LC_CTYPE en_US.UTF-8
+
+# frontend dependencies
+COPY yarn.lock package.json .yarnrc ./
+RUN yarn
```

Comment:
I suggest `yarn install --frozen-lockfile` for [reproducibility](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-frozen-lockfile).